<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimingChartForm</class>
 <widget class="QWidget" name="TimingChartForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timing</string>
  </property>
  <widget class="QScrollArea" name="scrollAreaTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>24</y>
     <width>191</width>
     <height>541</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>189</width>
      <height>539</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>741</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTimeUnit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1ms</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10ms</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100ms</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1s</string>
    </property>
   </item>
  </widget>
  <widget class="QScrollArea" name="scrollAreaSignal">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>24</y>
     <width>741</width>
     <height>541</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>739</width>
      <height>539</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="toolButtonSampling">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Sampling</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frameOperation">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>566</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButtonUpdate">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>0</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
